vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-testbench/src/main/java/com/vaadin/flow/component/datetimepicker/testbench/DateTimePickerElement.java

```diff
@@ -247,14 +249,24 @@ public boolean isAutoOpen() {
         return !getPropertyBoolean("autoOpenDisabled");
     }
 
-    private TestBenchElement getDatePicker() {
-        return $("vaadin-date-picker").withAttribute("slot", "date-picker")
-                .first();
+    /**
+     * Get the DatePickerElement for date picker part.
+     *
+     * @return DatePickerElement
+     */
+    public DatePickerElement getDatePicker() {
+        return this.$("vaadin-date-picker").withAttribute("slot", "date-picker")
+                .first().wrap(DatePickerElement.class);
```

Comment:
DateTimePicker is expected to have only one DatePicker.

```suggestion
        return this.$(DatePickerElement.class).first();
```